### PR TITLE
Update revenue_report.md

### PR DESCRIPTION
**Pull request by sambellbraze**
We think there is a huge flaw in our docs here where we don't explain at all the calculations for the top 3 graphs on the Revenue Report page. 

We've had 3 x Snr CSMs spend 1hr scouring the docs and trying to figure out where on earth all these metrics are coming from and if they are unique and de-duplicated outside of the campaigns and Canvases. 

We think the docs has to be updated to have this information we've inserted as it's now crystal clear (happy for you to fix our proposal copy up) what each of those metrics is.

# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @internetisaiah as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Ana Teresa Serafino on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @internetisaiah for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@internetisaiah</td>
    <td>Developer Guide<br>Technology partners<br>Channels</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>Frontend Infrastructure & Experience (FIX)</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/reporting/revenue_report.md [hunk 1]
```diff
@@ -41,6 +41,14 @@ You can view the following data in the **Performance Over Time Graph**:
     - (Optional) Revenue per Hour By Segment
 - Revenue per User
 
+## Understanding revenue calculations
+
+- *Lifetime revenue* is the total `PurchaseEvents` `price` value (in USD) received since inception. 
+- *Lifetime value per user* is the *Lifetime revenue* divided by your total *Users* (located on your **Home** page).
+- Average *Daily revenue* is de-duplicated from campaigns and Canvases.
+- Average *Daily Purchases* is the total unique `PurchaseEvents` received each day.
```
Comment (ATremonte) [inline _docs/_user_guide/data_and_analytics/reporting/revenue_report.md:49]: The definitions all look correct here. Should this line say it's the _average_ of "the total unique `PurchaseEvents`" over the time period?

File: _docs/_user_guide/data_and_analytics/reporting/revenue_report.md [hunk 2]
```diff
@@ -41,6 +41,14 @@ You can view the following data in the **Performance Over Time Graph**:
     - (Optional) Revenue per Hour By Segment
 - Revenue per User
 
+## Understanding revenue calculations
+
+- *Lifetime revenue* is the total `PurchaseEvents` `price` value (in USD) received since inception. 
+- *Lifetime value per user* is the *Lifetime revenue* divided by your total *Users* (located on your **Home** page).
+- Average *Daily revenue* is de-duplicated from campaigns and Canvases.
```
Comment (ATremonte) [inline _docs/_user_guide/data_and_analytics/reporting/revenue_report.md:48]: This is correct, though do customers know what "de-duplicated" means in this context? We could just say "daily revenue" is the sum of the campaign and canvas revenue for a given day

**Review by ATremonte — APPROVED** [review]

File: _docs/_user_guide/data_and_analytics/reporting/revenue_report.md (hunk 2 above)
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/reporting/revenue_report.md:48]:
```suggestion
- Average *Daily revenue* is the sum of the campaign and Canvas revenue for a given day.
```

File: _docs/_user_guide/data_and_analytics/reporting/revenue_report.md (hunk 1 above)
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/reporting/revenue_report.md:49]:
```suggestion
- *Daily Purchases* is the average of the total unique `PurchaseEvents` over the time period.
```

**Review by rachel-feinberg — APPROVED** [review]

**Pull request merged**